<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NpcManager</class>
 <widget class="QWidget" name="NpcManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>451</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabGenerate">
    <attribute name="title">
     <string>NPC Generation</string>
    </attribute>
    <widget class="QLineEdit" name="txtNpcYPos">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>91</x>
       <y>160</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>y pos</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNpcYRot">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>160</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>y rot</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbxGenNpcPosRot">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>130</y>
       <width>391</width>
       <height>21</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Generate an NPC from this specific position and rotation.
Keep in mind if you have &quot;Generate an NPC every...&quot; checked, it will start using these positions as it's origin instead of your pony's position.</string>
     </property>
     <property name="toolTipDuration">
      <number>15</number>
     </property>
     <property name="text">
      <string>Generate an NPC on this position and rotation</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNpcName">
     <property name="geometry">
      <rect>
       <x>91</x>
       <y>40</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This DOES NOT apply to the name above the pony's head, that is controlled by ponyData, this only applies to the npc filename, name metadata, and the npc name box in the dialog gui in-game when you talk to it.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="toolTipDuration">
      <number>20</number>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>NPC's Name. Leave Blank to use your pony's name.</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNpcXRot">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>211</x>
       <y>160</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>x rot</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNpcPonyData">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>371</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="placeholderText">
      <string>Get a pony's ponyData by pressing 'Get ponyData' in the general tab</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbxGenNPCself">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Generate an NPC of your pony in the exact position you are standing on. Does not include weared items.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="toolTipDuration">
      <number>15</number>
     </property>
     <property name="text">
      <string>Generate NPC of yourself at the current position</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNpcZRot">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>311</x>
       <y>160</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>z rot</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtItemIds">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>220</y>
       <width>371</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="placeholderText">
      <string>ItemId ItemId ItemId ...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNpcXPos">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>160</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>x pos</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbxGenNpcAccs">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>190</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Generate an NPC with the accesories you want, like a hat or something. Type in the item IDs in the box, seperate each with a space.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;To find a list of item IDs check out the /data/data/items.xml file.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="toolTipDuration">
      <number>15</number>
     </property>
     <property name="text">
      <string>Generate an NPC with these accesories:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>40</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>NPC Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNpcWRot">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>361</x>
       <y>160</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>w rot</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbxGenNpcPonyData">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>70</y>
       <width>391</width>
       <height>21</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This will generate NPCs based on the other of ponyDatas you put in.&lt;/p&gt;&lt;p&gt;This will only work if you have checked &amp;quot;Generate an NPC every...&amp;quot;, otherwise, it will generate an NPC with the first ponyData in the textbox.&lt;/p&gt;&lt;p&gt;If empty, it will use the ponyData of your current pony that you are playing as.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="toolTipDuration">
      <number>15</number>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Generate an NPC with this ponyData</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNpcZPos">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>151</x>
       <y>160</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>z pos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnGenerateNpc">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;You may need to reload the scene two times for your generated npc(s) to appear, if it still doesn't, try stopping the server and then starting it.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="toolTipDuration">
      <number>20</number>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Generate NPC</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>NPC Live Edit</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>391</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Select NPC loaded from the current Set Player's Scene</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>421</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>411</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>No NPC in memory.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>321</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>NPC PonyData</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>128</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>NPC Position</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>180</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>180</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>180</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>NPC Rotation</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>230</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>230</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>230</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_8">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>230</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>260</y>
       <width>421</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>391</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;When selecting an NPC, that NPC's information will be stored in memory and&lt;br/&gt; then unloaded from the scene, then loading the one in memory after a scene&lt;br/&gt;refresh, ready to be edited.&lt;/p&gt;&lt;p&gt;You can only edit the information above, as they are the only ones used for&lt;br/&gt;the making of Machinimas.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>30</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Use</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
